source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -513,7 +511,7 @@ def _datetime_to_str(ts):
             dep=_datetime_to_str(journey.departure_date_time),
             arr=_datetime_to_str(journey.arrival_date_time),
             duration=datetime.timedelta(seconds=journey.duration),
-            fallback=datetime.timedelta(seconds=fallback_duration(journey, 0)),
+            fallback=datetime.timedelta(seconds=fallback_duration(journey)),
```

Comment:
So the debug fallback walking duration will be back to not considering transfers.
No major issue on that.